src/aequitas/bias.py

```diff
@@ -156,19 +165,21 @@ def get_disparity_min_metric(self, df, original_df, key_columns=None,
 
             # check what new disparity columns are and order as disparity,
             # ref_group, significance for each
+            base_sig = [sig for sig in ['label_value_significance', 'score_significance'] if
+                        ''.join(sig.split('_significance')) in selected_significance]
```

Comment:
this seems like a very awkward use of `split()`

why not:

```suggestion
                        sig.replace('_significance', '') in selected_significance]
```